ts/indicators/indicators.src.ts

```diff
@@ -22,7 +22,7 @@ declare global {
             public data: Array<SMAIndicatorPoint>;
             public dataEventsToUnbind: Array<Function>;
             public hasDerivedData: boolean;
-            public linkedParent: Series;
+            public linkedParent: LineSeries;
```

Comment:
I'm not sure about this one - does `LineSeries` cover data format from OHLCSeries? For example SMA indicator can work with all possible linked parents (Line, Range, OHLC/Candlestick).

Reply:
Ok, then it should probably still use `Series`. Then I assume we need to change the generic type to extend from `Series`and not `LineSeries`as well?

Reply:
I think so. Any thoughts @bre1470 ?

Reply:
Practically both should be the same, but in type sense it might be better to rather use `Series` than `LineSeries`.

Reply:
:+1: Ok, I'll change it

Reply:
Changed to Series. FYI @pawelfus @bre1470

Reply:
Thanks!